Scripts/script-StopTimeToAssignOnOwnerChange.yml

```diff
@@ -0,0 +1,24 @@
+fromversion: 4.1.0
+commonfields:
+  id: stoptimetoassignonownerchange
+  version: -1
+name: StopTimeToAssignOnOwnerChange
+script: |
+  # ##### Help #####
+  # This is an example script. The script is used to stop the Time to Assignment SLA field, once an owner was set to an incident.
+  # If you want to use this script, you should go to the Time to Assignment SLA field, and set this script as the script to run upon change of field value.
+
+  if demisto.args()['old'] == "" and demisto.args()['new'] != "":  # If owner was no-one and is now someone:
```

Comment:
Good idea

Reply:
Actually, turns out it still gives "keyerror" because 'old' is not False, it's simply inexistent.
The solution is, however, to use if not demisto.args().get('old'). using `get` will return None if the key doesn't exist, without raising an error.

Reply:
You are correct